src/app/components/rows/NewRow.jsx

```diff
@@ -22,16 +22,14 @@ const withFunctionality = compose(
   withHandlers({
     addNewRow: props => () => {
       const { t, showToast, rows, onAdd } = props;
-      const isEmpty = stuff => {
-        const res = f.cond([
-          [
-            f.isArray,
-            element => f.isEmpty(element) || f.every(f.isEmpty, element)
-          ],
-          [f.stubTrue, f.isEmpty]
-        ])(stuff);
-        return res;
-      };
+      const isEmpty = f.cond([
+        [
+          f.isArray,
+          element => f.isEmpty(element) || f.every(f.isEmpty, element)
```

Comment:
```suggestion
          element => f.isEmpty(element) || f.every(isEmpty, element)
```
Sonst ist ein Array aus Zahlen auch leer, wodurch der ganze Sinn der Funktion hinüber ist...